components/dialog/integration.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Integration
+page_title: Dialog Integration
+description: Discover the Blazor Dialog Integration and explore the examples.
+slug: dialog-integration
+tags: telerik,blazor,dialog,integration
+published: True
+position: 9
+---
+
+# Blazor Dialog Integration
+
+You can use other components inside the `DialogContent`. In this case, the two-way binding for the parameters of these components may not work as expected. The Dialog needs to [`Refresh`]({%slug dialog-overview%}#dialog-reference-and-methods) to reflect UI changes.
+
+This article contains the following examples:
+
+* [Checkbox integration in Dialog](#checkbox-in-a-dialog)
+* [Filter integration in Dialog](#filter-in-a-dialog)
+
+## Checkbox in a Dialog
+
+**To integrate the Checkbox in the Dialog, you can:**
```

Comment:
Avoid using bold unless prescribed by the [guidelines ](https://docs.telerik.com/style-guide/element-formatting#guidelines)in the Style Guide.
```suggestion
To integrate the Checkbox in the Dialog:
```